api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java

```diff
@@ -436,4 +363,65 @@ private DbConceptSet getDbConceptSet(
     }
     return conceptSet;
   }
+
+  public static ConceptSet toClientConceptSet(DbConceptSet dbConceptSet) {
```

Comment:
IYI, MapStruct may be able to help here. It generates the obvious fields and lets your specify the nontrivial stuff.